pkg/custompluginmonitor/plugin/plugin.go

```diff
@@ -127,16 +128,21 @@ func (p *Plugin) run(rule cpmtypes.CustomRule) (exitStatus cpmtypes.Status, outp
 	defer cancel()
 
 	cmd := exec.CommandContext(ctx, rule.Path, rule.Args...)
-	stdout, err := cmd.Output()
-	if err != nil {
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
```

Comment:
I have some concern about this, especially stderr. If the stderr is too big, it could potentially cause high memory usage.

Can we implement a limit reader to limit the output size, and discard other outputs?

Reply:
Added a limit reader for stdout and stderr limiting each to 4k bytes.